src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -295,6 +305,18 @@ public function configureOptions(OptionsResolver $resolver)
             return $options['expanded'];
         };
 
+        $choiceTranslationDomain = function (Options $options) {
+            return $options['translation_domain'];
+        };
+
+        $choiceTranslationDomainNormalizer = function (Options $options, $choiceTranslationDomain) {
+            if (true === $choiceTranslationDomain) {
+                return;
```

Comment:
yep, lets do it this way:
- true: reuse translation_domain 
- false: disabled
- null: leave untouched (default domain `messages`)
- explicit string